specification/schemas/Hook.json

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "object",
+  "required": [
+    "resource_type",
+    "resource_action",
+    "predicate",
+    "action"
+  ],
+  "properties": {
+    "resource_type": {
+      "type": "string",
+      "example": "services"
+    },
+    "resource_action": {
+      "type": "string",
+      "enum": [
+        "create",
+        "read",
+        "update",
+        "delete"
```

Comment:
We're only going to support `create` and `update` initially. So for now I think it's best to remove the `delete` and `update` resource actions, to keep people from trying to create hooks that don't work (yet).